### PR TITLE
added visualbar to similarity column

### LINKED ISSUE #95: Visual indicator of how similar utterances are in the similar utterances tab
Bars? With different colors when the similarity is above or below the threshold (0.5) that determine the `no_close` smart tag?

### PR DESCRIPTION
**Pull request by nandhinibsn**
Resolve #95

## Checklist:

You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge it.

* [x] **ISSUE NUMBER.** You linked the issue number (Ex: Resolve #XXX).
* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **FRONTEND TYPES.** Regenerate the front-ent types if you played with types and routes.
  Run `cd webapp && yarn types` while the back-end is running.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

**Comment by nandhinibsn** [comment]
Adde screenshot of the change
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Released changes are shown in the
 
 ### Added
 - Rows of Performance Analysis table now link to exploration page with filters applied.
+- Added a distribution bar for the similarity column in the semantically similar utterances.
```
Comment (JosephMarinier) [inline CHANGELOG.md:15]: It's not really a "distribution bar". We called it a "visual bar" in the previous release, so for consistency I suggest
```suggestion
- Added visual bars to the similarity column in the semantically similar utterances.
```

**Review by JosephMarinier — APPROVED** [review]
Cool! Two tiny comments, but that will be good to go! 👍

File: webapp/src/components/Metrics/PerformanceAnalysis.tsx [hunk 2]
```diff
@@ -11,7 +11,6 @@ import {
   GridColumnMenuContainer,
   GridColumnMenuProps,
   GridColumnsMenuItem,
-  GridColumnVisibilityModel,
```
[the remark this answers is not in the document]
Comment (nandhinibsn) [reply webapp/src/components/Metrics/PerformanceAnalysis.tsx:14]: okay I don't see thats getting used in the code so removed it. is it for the recent column visibility feature? If so, Sure, I will revert back...

**Comment by nandhinibsn** [comment]
Thank for reviewing it @JosephMarinier :)

File: webapp/src/components/Metrics/PerformanceAnalysis.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (JosephMarinier) [reply webapp/src/components/Metrics/PerformanceAnalysis.tsx]: Yes. Again, if you merge with `main`, you should see it is used.

**Pull request merged**